frontend/src/hooks/useGlobalLocalStorage.ts

```diff
@@ -11,8 +11,13 @@ export const useGlobalLocalStorage = () => {
         {},
     );
 
+    const parsedValue = {
```

Comment:
This fixes reading incorrect values that we already put into local storage. All users of this code will get this fix without changing each file